### PR TITLE
[ADD] saas_sysadmin_mailgun: new module

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

**Comment by iledarn** [comment]
@yelizariev, our plan in doc/index.rst

File: saas_sysadmin_mailgun/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+==========================
+
+Usage
+=====
+
+* Put Mailgun API Key in Settings > Saas Portal Settings
+* On create new client database a new mail domain will be created on mailgun
+* Mail domain is validated using dns records through AWS Route53
+* In the client database Incoming and Outgoing mail configuration will be done
+* To receive mails there should be mail-addons/mail_mailgun module installed in the client database
```
Comment (yelizariev) [inline saas_sysadmin_mailgun/doc/index.rst:16]: say instead, that mailgun must be installed in template database

Comment (yelizariev) [inline saas_sysadmin_mailgun/doc/index.rst:16]: give a link to mail_mailgun module

File: saas_sysadmin_mailgun/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Saas Sysadmin Mailgun",
+    'author': "IT-Projects LLC, Ildar Nasyrov",
+    'license': 'LGPL-3',
+    'website': "https://twitter.com/nasyrov_ildar",
+    'category': 'SaaS',
+    'version': '1.0.0',
+    'depends': ['saas_sysadmin'],
```
Comment (yelizariev) [inline saas_sysadmin_mailgun/__openerp__.py]: add saas_sysadmin_route53

Comment (yelizariev) [inline saas_sysadmin_mailgun/__openerp__.py]: or new module, say  saas_sysadmin_aws for credentials

**Comment by iledarn** [comment]
@yelizariev,
need your help

**Comment by iledarn** [comment]
@yelizariev ,
need your help

File: saas_sysadmin_mailgun/doc/index.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+==========================
+ Mailgun for saas clients
+==========================
+
+Usage
+=====
+
+* Put Mailgun API Key in Settings > Saas Portal Settings
```
Comment (yelizariev) [inline saas_sysadmin_mailgun/doc/index.rst]: add info how to configure route53 credentials

**Comment by iledarn** [comment]
Two new modules added:
- saas_sysadmin_aws_route53
- saas_sysadmin_mailgun

**Comment by iledarn** [comment]
@yelizariev

File: saas_sysadmin_mailgun/models/mailgun.py [hunk 4]
```diff
@@ -192,3 +200,14 @@ def create_credentials(api_key=None, domain=None):
         data={"login": "alice@YOUR_DOMAIN_NAME",
               "password": "secret"})
 
+def create_store_route(api_key=None, domain=None, mail_domain=None):
+    """Create a route for message storing and notification"""
+    action = "store(notify='http://%s/mailgun/notify')" % domain
```
Comment (yelizariev) [inline saas_sysadmin_mailgun/models/mailgun.py]: It must work with https too

**Pull request merged**